starter-code/index.html

```diff
@@ -2,29 +2,76 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css" />
 </head>
 <body>
-Premium Discover Help Download
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
+<div class = "banner">
+ <div> <!-- spotify image -->
+  <img class = "img-size col" src= "./images/spotify-logo.png"/>
+ </div>
+ 
+ <div>
+ <h3 class = "col-1">Premium</h3> 
+ <h3 class = "col-1">Discover</h3> 
+ <h3 class = "col-1">Help</h3> 
+ <h3 class = "col-1">Download</h3> 
 
-What’s on Spotify?
+ </div>
 
-Millions of Songs
-There are millions of
-songs on Spotify
+</div> <!-- end of class banner -->
 
-HD Music
-Listen to music as if you
-were listening live
+<div class = "main">
+  
+<h1 >Music for everyone.</h1>
+<div>
+  <h3>Spotify is now free on mobile, tablet and computer.<br/>
+  Listen to the right music, wherever you are.</h3>
+</div>
 
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
+
+</div> <!-- end of class main -->
+
+<div class = "bottom">
+
+<div class="center" > <h2 class = "underline" >What’s on Spotify?</h2></div>
+
+<div class="box col">
+  <img src= "./images/music-icon.png"/>
+  <h5>Millions of Songs</h5>
+  <p>There are millions of
+  songs on Spotify</p>
+</div>
+
+<div class = "box col">
+  <img class = "img-size" src= "./images/high-quality-icon.png"/>
+  <h5>HD Music</h5>
+  <p>Listen to music as if you
+  were listening live</p>
+</div>
+
+
+<div class="box col">
+  <img src= "./images/devices-icon.png"/>
+  <h5>Stream Everywhere</h5>
+  <p>Stream music on your
+  smartphone, tablet or
+  computer</p>
+</div>
+<div class="clearfix"></div>
+
+</div> <!-- end of bottom class -->
+
+
+
+
+
+
+
+<!-- for now, stop here -->
 
 It’s as yeezy as Kanye West.
```

Comment:
Don't forget to put your content inside tag (p, div, ect..) otherwise it will randomly appear on your page and you want be able to style it ;)